### PR TITLE
Add chain Nexi

### PR DESCRIPTION
**Pull request by polygomic**
## Description

_Concise description of proposed changes_

## Additional Information

- [x] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

Your ENS/address:   0x06aD7D3FB4de5302f7659aB9455541D3c88786A2

### CODE REVIEW

**Comment by polygomic** [comment]
Please add chain nexi

File: .changeset/large-eels-roll.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@wagmi/chains': minor
```
Comment (tmm) [inline .changeset/large-eels-roll.md:2]:
```suggestion
'@wagmi/chains': patch
```

File: .changeset/large-eels-roll.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+Add chain Nexi
```
Comment (tmm) [inline .changeset/large-eels-roll.md:5]:
```suggestion
Added Nexi Chain
```

File: yarn.lock
Comment (tmm) [inline yarn.lock:1]: remove this please

Comment (polygomic) [reply yarn.lock:1]: bug has been fixed.

Comment (polygomic) [reply yarn.lock:1]: Yarn.lock has been deleted.

File: packages/chains/src/nexi.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { Chain } from './types'
+
+export const nexi = {
+  id: 4242,
```
Comment (tmm) [inline packages/chains/src/nexi.ts:4]:
```suggestion
  id: 4_242,
```

**Review by tmm — APPROVED** [review]

**Pull request merged**